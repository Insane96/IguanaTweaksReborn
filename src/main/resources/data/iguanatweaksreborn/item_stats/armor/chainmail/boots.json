{
	"item": "minecraft:chainmail_boots",
	"durability": 48,
	"armor": 1,
	"toughness": 1
}