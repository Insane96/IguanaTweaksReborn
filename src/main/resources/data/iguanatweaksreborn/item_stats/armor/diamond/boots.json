{
	"item": "minecraft:diamond_boots",
	"durability": 180,
	"armor": 1,
	"toughness": 1,
	"knockback_resistance": 0.03
}