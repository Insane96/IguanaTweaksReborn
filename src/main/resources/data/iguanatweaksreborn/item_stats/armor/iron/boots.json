{
	"item": "minecraft:iron_boots",
	"durability": 72,
	"armor": 2
}