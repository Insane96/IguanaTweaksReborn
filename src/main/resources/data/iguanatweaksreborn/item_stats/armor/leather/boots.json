{
	"item": "minecraft:leather_boots",
	"durability": 72,
	"armor": 1,
	"toughness": 1
}