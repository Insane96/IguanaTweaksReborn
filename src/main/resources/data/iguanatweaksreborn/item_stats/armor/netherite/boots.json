{
	"item": "minecraft:netherite_boots",
	"durability": 168,
	"armor": 3,
	"armor_toughness": 2,
	"knockback_resistance": 0.05
}