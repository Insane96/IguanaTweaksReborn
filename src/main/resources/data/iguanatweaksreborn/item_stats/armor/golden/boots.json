{
	"item": "minecraft:golden_boots",
	"durability": 72,
	"armor": 1,
	"toughness": 1,
	"modifiers": [
		{
			"uuid": "dc49f564-489f-4f70-ab50-ce85cc4bfa85",
			"name": "Golden armor bonus max health",
			"slot": "feet",
			"attribute": "minecraft:generic.max_health",
			"amount": 2,
			"operation": "addition"
		}
	]
}